x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -465,81 +503,76 @@ const ProcessEventDotComponents = React.memo(
               </use>
             </g>
           </svg>
-          <div
-            style={{
-              display: 'flex',
-              flexFlow: 'column',
-              left: '25%',
-              top: '30%',
-              position: 'absolute',
-              width: '50%',
-              color: NamedColors.full,
-              fontSize: `${scaledTypeSize}px`,
-              lineHeight: '140%',
-              backgroundColor: NamedColors.resolverBackground,
-              padding: '.25rem',
-            }}
+          <StyledActionsContainer
+            color={colorMap.full}
+            fontSize={scaledTypeSize}
+            topPct={actionsYOffsetPct}
           >
+            {isShowingDescriptionText && (
+              <StyledDescriptionText
+                backgroundColor={colorMap.resolverBackground}
+                color={colorMap.descriptionText}
+              >
+                {descriptionText}
+              </StyledDescriptionText>
+            )}
             <div
-              id={descriptionId}
-              style={{
-                textTransform: 'uppercase',
-                letterSpacing: '-0.01px',
-                backgroundColor: NamedColors.resolverBackground,
-                lineHeight: '1',
-                fontWeight: 'bold',
-                fontSize: '0.8rem',
-                width: '100%',
-                margin: '0',
-                textAlign: 'left',
-                padding: '0',
-                color: NamedColors.empty,
-              }}
-            >
-              {descriptionText}
-            </div>
-            <div
-              className={magFactorX >= 2 ? 'euiButton' : 'euiButton euiButton--small'}
-              data-test-subject="nodeLabel"
-              id={labelId}
               style={{
-                backgroundColor: labelBackground,
-                padding: '.15rem 0',
-                textAlign: 'center',
-                maxWidth: '20rem',
-                minWidth: '12rem',
-                width: '60%',
-                overflow: 'hidden',
-                whiteSpace: 'nowrap',
-                textOverflow: 'ellipsis',
-                contain: 'content',
-                margin: '.25rem 0 .35rem 0',
+                backgroundColor: colorMap.resolverBackground,
+                alignSelf: 'flex-start',
+                padding: 0,
               }}
             >
-              <span className="euiButton__content">
-                <span className="euiButton__text" data-test-subj={'euiButton__text'}>
-                  {eventModel.eventName(event)}
+              <EuiButton
+                color={labelButtonFill}
+                data-test-subject="nodeLabel"
+                fill={isLabelFilled}
+                id={labelId}
+                size="s"
+                style={{
+                  maxHeight: `${Math.min(26 + magFactorX * 3, 32)}px`,
```

Comment:
can you explain this expression?

Reply:
Keeping the max-height of the button at 32, but it allows the sides of the button to better line up with the edge of the processNode